cmd/admin-revoker/main.go

```diff
@@ -24,23 +22,33 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
-func loadConfig(c *cli.Context) (config cmd.Config, err error) {
-	configFileName := c.GlobalString("config")
-	configJSON, err := ioutil.ReadFile(configFileName)
-	if err != nil {
-		return
-	}
+const clientName = "AdminRevoker"
```

Comment:
Ack, I was just searching within GitHub's pull request page, didn't realize this was used in a hidden fold.